### ISSUE TITLE
Accounts app Dev Environment: Minor UI issue with Show button

### ISSUE THREAD

**Issue by nithyaasworld** [topcoder-platform/community-app#3298]
Steps to Reproduce:
1. Enter invalid username and password
2. Check the Show button UI

Actual:
1. Bottom line of Show button is cut
2. Grey fill for Show button is missing

![image](https://user-images.githubusercontent.com/5114190/64452682-7d134500-d104-11e9-915c-cca25ed12d3c.png)

**Comment by veshu** [comment]
@nithyaasworld I can't reproduce the cut off issue. 
![image](https://user-images.githubusercontent.com/2202045/64508776-ef02ae80-d2fd-11e9-8d56-eee14356fbbe.png)


Grey color is intentionally removed.

**Comment by nithyaasworld** [comment]
@veshu Weird! It's always reproducible for me. Please find the video here.


[Show button underline issue.zip](https://github.com/topcoder-platform/community-app/files/3589068/Show.button.underline.issue.zip)

**Comment by nithyaasworld** [comment]
@veshu As its not reproducible to both of you and @r0hit-gupta and the issue is not something to worry about, I will close this issue. 

Thanks @veshu @r0hit-gupta for checking!

CC: @sushilshinde
